### ISSUE TITLE
2024-12-07, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#13]
# Today's Arxiv Papers

(1) [p-MoD: Building Mixture-of-Depths MLLMs via Progressive Ratio Decay](http://arxiv.org/abs/2412.04449v1), 2024-12-05

(2) [EgoPlan-Bench2: A Benchmark for Multimodal Large Language Model Planning in Real-World Scenarios](http://arxiv.org/abs/2412.04447v1), 2024-12-05

(3) [Moto: Latent Motion Token as the Bridging Language for Robot Manipulation](http://arxiv.org/abs/2412.04445v1), 2024-12-05

(4) [Divot: Diffusion Powers Video Tokenizer for Comprehension and Generation](http://arxiv.org/abs/2412.04432v1), 2024-12-05

(5) [Grounding Descriptions in Images informs Zero-Shot Visual Recognition](http://arxiv.org/abs/2412.04429v1), 2024-12-05

(6) [Florence-VL: Enhancing Vision-Language Models with Generative Vision Encoder and Depth-Breadth Fusion](http://arxiv.org/abs/2412.04424v1), 2024-12-05

(7) [Targeting the Core: A Simple and Effective Method to Attack RAG-based Agents via Direct LLM Manipulation](http://arxiv.org/abs/2412.04415v1), 2024-12-05

(8) [Discriminative Fine-tuning of LVLMs](http://arxiv.org/abs/2412.04378v1), 2024-12-05

(9) [Challenges in Trustworthy Human Evaluation of Chatbots](http://arxiv.org/abs/2412.04363v1), 2024-12-05

(10) [RMD: A Simple Baseline for More General Human Motion Generation via Training-free Retrieval-Augmented Motion Diffuse](http://arxiv.org/abs/2412.04343v1), 2024-12-05

(11) [Retrieval-Augmented Machine Translation with Unstructured Knowledge](http://arxiv.org/abs/2412.04342v1), 2024-12-05

(12) [Liquid: Language Models are Scalable Multi-modal Generators](http://arxiv.org/abs/2412.04332v1), 2024-12-05

(13) [The Hyperfitting Phenomenon: Sharpening and Stabilizing LLMs for Open-Ended Text Generation](http://arxiv.org/abs/2412.04318v1), 2024-12-05

(14) [FlashSloth: Lightning Multimodal Large Language Models via Embedded Visual Compression](http://arxiv.org/abs/2412.04317v1), 2024-12-05

(15) [Densing Law of LLMs](http://arxiv.org/abs/2412.04315v1), 2024-12-05

(16) [ALMA: Alignment with Minimal Annotation](http://arxiv.org/abs/2412.04305v1), 2024-12-05

(17) [Evolutionary Pre-Prompt Optimization for Mathematical Reasoning](http://arxiv.org/abs/2412.04291v1), 2024-12-05

(18) [Arabic Stable LM: Adapting Stable LM 2 1.6B to Arabic](http://arxiv.org/abs/2412.04277v1), 2024-12-05

(19) [PoTable: Programming Standardly on Table-based Reasoning Like a Human Analyst](http://arxiv.org/abs/2412.04272v1), 2024-12-05

(20) [SynFinTabs: A Dataset of Synthetic Financial Tables for Information and Table Extraction](http://arxiv.org/abs/2412.04262v1), 2024-12-05

(21) [VASCAR: Content-Aware Layout Generation via Visual-Aware Self-Correction](http://arxiv.org/abs/2412.04237v1), 2024-12-05

(22) [Addressing Hallucinations with RAG and NMISS in Italian Healthcare LLM Chatbots](http://arxiv.org/abs/2412.04235v1), 2024-12-05

(23) [A Context-aware Framework for Translation-mediated Conversations](http://arxiv.org/abs/2412.04205v1), 2024-12-05

(24) [AL-QASIDA: Analyzing LLM Quality and Accuracy Systematically in Dialectal Arabic](http://arxiv.org/abs/2412.04193v1), 2024-12-05

(25) [Leveraging Large Language Models to Generate Course-specific Semantically Annotated Learning Objects](http://arxiv.org/abs/2412.04185v1), 2024-12-05

(26) [SKIM: Any-bit Quantization Pushing The Limits of Post-Training Quantization](http://arxiv.org/abs/2412.04180v1), 2024-12-05

(27) [Bench-CoE: a Framework for Collaboration of Experts from Benchmark](http://arxiv.org/abs/2412.04167v1), 2024-12-05

(28) [Reducing Tool Hallucination via Reliability Alignment](http://arxiv.org/abs/2412.04141v1), 2024-12-05

(29) [Monet: Mixture of Monosemantic Experts for Transformers](http://arxiv.org/abs/2412.04139v1), 2024-12-05

(30) [Enhancing Mathematical Reasoning in LLMs with Background Operators](http://arxiv.org/abs/2412.04110v1), 2024-12-05

(31) [Pre-train, Align, and Disentangle: Empowering Sequential Recommendation with Large Language Models](http://arxiv.org/abs/2412.04107v1), 2024-12-05

(32) [Practical Considerations for Agentic LLM Systems](http://arxiv.org/abs/2412.04093v1), 2024-12-05

(33) [LossAgent: Towards Any Optimization Objectives for Image Processing with LLM Agents](http://arxiv.org/abs/2412.04090v1), 2024-12-05

(34) [Towards Generalizable Autonomous Penetration Testing via Domain Randomization and Meta-Reinforcement Learning](http://arxiv.org/abs/2412.04078v1), 2024-12-05

(35) [Does your model understand genes? A benchmark of gene properties for biological and text models](http://arxiv.org/abs/2412.04075v1), 2024-12-05

(36) [ProtDAT: A Unified Framework for Protein Sequence Design from Any Protein Text Description](http://arxiv.org/abs/2412.04069v1), 2024-12-05

(37) [From Code to Play: Benchmarking Program Search for Games Using Large Language Models](http://arxiv.org/abs/2412.04057v1), 2024-12-05

(38) [Prompt Engineering Guidance for Conceptual Agent-based Model Extraction using Large Language Models](http://arxiv.org/abs/2412.04056v1), 2024-12-05

(39) [Hostility Detection in UK Politics: A Dataset on Online Abuse Targeting MPs](http://arxiv.org/abs/2412.04046v1), 2024-12-05

(40) [GenChaR: A Dataset for Stock Chart Captioning](http://arxiv.org/abs/2412.04041v1), 2024-12-05

(41) [SocialMind: LLM-based Proactive AR Social Assistive System with Human-like Perception for In-situ Live Interactions](http://arxiv.org/abs/2412.04036v1), 2024-12-05

(42) [Marco-LLM: Bridging Languages via Massive Multilingual Training for Cross-Lingual Enhancement](http://arxiv.org/abs/2412.04003v1), 2024-12-05

(43) [MTMT: Consolidating Multiple Thinking Modes to Form a Thought Tree for Strengthening LLM](http://arxiv.org/abs/2412.03987v1), 2024-12-05

(44) [Comprehensive Audio Query Handling System with Integrated Expert Models and Contextual Understanding](http://arxiv.org/abs/2412.03980v1), 2024-12-05

(45) [AI-based Attacker Models for Enhancing Multi-Stage Cyberattack Simulations in Smart Grids Using Co-Simulation Environments](http://dx.doi.org/10.1109/CSR61664.2024.10679448), 2024-12-05

(46) [Demonstration Selection for In-Context Learning via Reinforcement Learning](http://arxiv.org/abs/2412.03966v1), 2024-12-05

(47) [Chain-of-Thought in Large Language Models: Decoding, Projection, and Activation](http://arxiv.org/abs/2412.03944v1), 2024-12-05

(48) [AIpparel: A Large Multimodal Generative Model for Digital Garments](http://arxiv.org/abs/2412.03937v1), 2024-12-05

(49) [Exploring AI Text Generation, Retrieval-Augmented Generation, and Detection Technologies: a Comprehensive Overview](http://arxiv.org/abs/2412.03933v1), 2024-12-05

(50) [A Survey on Large Language Model-Based Social Agents in Game-Theoretic Scenarios](http://arxiv.org/abs/2412.03920v1), 2024-12-05

(51) [Integrating Various Software Artifacts for Better LLM-based Bug Localization and Program Repair](http://arxiv.org/abs/2412.03905v1), 2024-12-05

(52) [MISR: Measuring Instrumental Self-Reasoning in Frontier Models](http://arxiv.org/abs/2412.03904v1), 2024-12-05

(53) [Uniform Discretized Integrated Gradients: An effective attribution based method for explaining large language models](http://arxiv.org/abs/2412.03886v1), 2024-12-05

(54) [CreatiLayout: Siamese Multimodal Diffusion Transformer for Creative Layout-to-Image Generation](http://arxiv.org/abs/2412.03859v1), 2024-12-05

(55) [How Good is ChatGPT in Giving Adaptive Guidance Using Knowledge Graphs in E-Learning Environments?](http://arxiv.org/abs/2412.03856v1), 2024-12-05

(56) [Educational-Psychological Dialogue Robot Based on Multi-Agent Collaboration](http://arxiv.org/abs/2412.03847v1), 2024-12-05

(57) [A large language model-type architecture for high-dimensional molecular potential energy surfaces](http://arxiv.org/abs/2412.03831v1), 2024-12-05

(58) [Beyond the Binary: Capturing Diverse Preferences With Reward Regularization](http://arxiv.org/abs/2412.03822v1), 2024-12-05

(59) [Synergizing LLMs and Knowledge Graphs: A Novel Approach to Software Repository-Related Question Answering](http://arxiv.org/abs/2412.03815v1), 2024-12-05

(60) [EditScout: Locating Forged Regions from Diffusion-based Edited Images with Multimodal LLM](http://arxiv.org/abs/2412.03809v1), 2024-12-05

(61) [Agent AI with LangGraph: A Modular Framework for Enhancing Machine Translation Using Large Language Models](http://arxiv.org/abs/2412.03801v1), 2024-12-05

(62) [Automated Multi-Label Annotation for Mental Health Illnesses Using Large Language Models](http://arxiv.org/abs/2412.03796v1), 2024-12-05
